x-pack/test/functional/apps/code/explore_repository.ts

```diff
@@ -268,7 +267,13 @@ export default function exploreRepositoryFunctionalTests({
         });
 
         // Click the structure tree tab
+        await retry.try(async () => {
+          expect(await testSubjects.isEnabled('codeStructureTreeTab')).to.be.ok();
```

Comment:
how this is different than line 266?

Reply:
you mean `expect(await exists('codeStructureTreeTab')).to.be(true);`?
This checks if structure tab is enabled, meaning structure tree data  is loaded.